assets/templates/login.html

```diff
@@ -1,46 +1,68 @@
-<!DOCTYPE html>
+  <!DOCTYPE html>
 <html lang="fr">
   <head>
     <meta charset="utf-8">
     <title>Cozy</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="/settings/theme.css">
-    <link rel="stylesheet" href="/assets/styles/stack.css">
+    <link rel="stylesheet" href="/assets/fonts/fonts.css">
+    <link rel="stylesheet" href="/assets/styles/login.css">
```

Comment:
My  goal was to separate the two concerns (login an authorization) until we update authorization, to avoid conflict and regression. When the authorization styles will be up-to-date, we will be able to merge all styles in one file. What do you think ?